### PR TITLE
Update release date in `HISTORY.rst`

### PR DESCRIPTION
**Pull request by prusse-martin**

### CODE REVIEW

**Review by nicoddemus — APPROVED** [review]

**Review by oliveira-mauricio — APPROVED** [review]

**Review by oliveira-mauricio — APPROVED** [review]

File: requirements_dev.txt [hunk 1]
```diff
@@ -7,6 +7,6 @@ pytest
 pytest-cov
 pytest-mock
 pytest-regressions
-strictyaml
 toposort
 typing_inspect
```
Comment (nicoddemus) [inline requirements_dev.txt:11]:
```suggestion
```

File: requirements_dev.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-barril
+barril>=1.10,<1.11
```
Comment (nicoddemus) [inline requirements_dev.txt:1]:
```suggestion
```

File: setup.cfg [hunk 3]
```diff
@@ -35,10 +35,12 @@ python_requires = >=3.6
...
+# TODO: #180 Temporary pin strictyaml (with ruamel.yaml).
 install_requires =
     python-hookman
     typing_inspect
-    strictyaml
+    strictyaml==1.3.2
+    ruamel.yaml==0.16.12
     boltons
     Click>=7.0
     barril>=1.9.0
```
Comment (nicoddemus) [inline setup.cfg:46]:
```suggestion
    barril>=1.10,<1.11
```

File: setup.cfg [hunk 4]
```diff
@@ -35,10 +35,12 @@ python_requires = >=3.6
 zip_safe = no
 setup_requires =
     setuptools>=40.0
+# TODO: #180 Temporary pin strictyaml (with ruamel.yaml).
 install_requires =
     python-hookman
     typing_inspect
-    strictyaml
+    strictyaml==1.3.2
```
Comment (nicoddemus) [inline setup.cfg:42]:
```suggestion
    strictyaml<1.4
```

File: setup.cfg [hunk 5]
```diff
@@ -35,10 +35,12 @@ python_requires = >=3.6
...
+    ruamel.yaml==0.16.12
```
Comment (nicoddemus) [inline setup.cfg:43]:
```suggestion
```

File: HISTORY.rst [hunk 6]
```diff
@@ -2,7 +2,13 @@
 History
 =======
 
-0.10.0 (unreleased)
+0.10.1 (2021-06-30)
+===================
+
+* Temporary pin strictyaml (dependency) due conflicts.
```
Comment (nicoddemus) [inline HISTORY.rst:8]:
```suggestion
* Temporarily pin ``strictyaml`` dependency due to conflicts.
```

File: setup.cfg [hunk 7]
```diff
@@ -35,10 +35,12 @@ python_requires = >=3.6
 zip_safe = no
 setup_requires =
     setuptools>=40.0
+# TODO: #180 Temporary pin strictyaml (with ruamel.yaml).
```
Comment (nicoddemus) [inline setup.cfg:38]:
```suggestion
```

**Review by nicoddemus — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 8]
```diff
@@ -38,8 +38,9 @@ setup_requires =
 install_requires =
     python-hookman
     typing_inspect
-    strictyaml
+    strictyaml<1.4
+    ruamel.yaml<0.16.13
```
Comment (prusse-martin) [inline setup.cfg:42]: I did return the ruamel pin, `0.16.13` was causing failures (aparentlly interface changes).
Note that strictyaml in `1.4.3` in now bundling ruamel `0.16.12`.

**Review by nicoddemus — APPROVED** [review]

**Pull request merged**